kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardHolderUpdateServiceImpl.java

```diff
@@ -0,0 +1,366 @@
+package edu.arizona.kfs.fp.batch.service.impl;
+
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.fp.businessobject.defaultvalue.NextProcurementCardDefaultIdFinder;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.kim.api.identity.PersonService;
+import org.kuali.rice.kim.api.identity.Person;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.rice.krad.util.ObjectUtils;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.arizona.kfs.fp.businessobject.ProcurementCardDefault;
+import edu.arizona.kfs.fp.businessobject.ProcurementCardHolderLoad;
+import edu.arizona.kfs.fp.batch.service.ProcurementCardHolderUpdateService;
+import edu.arizona.kfs.sys.KFSPropertyConstants;
+
+/**
+ * This is the default implementation of the ProcurementCardHolderUpdateService interface.
+ */
+@Transactional
+public class ProcurementCardHolderUpdateServiceImpl implements ProcurementCardHolderUpdateService {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ProcurementCardHolderUpdateServiceImpl.class);
+         
+    private BusinessObjectService businessObjectService;
+    private DateTimeService dateTimeService;
+    private Person cardholderUser;
+    
+    private static final String GO_LIVE_DATE = "2011-10-01";
+    private static final String UA = "UA";
+    private static final String OPEN = "Open";
+    private static final String DEFAULT_SYS_ID = "0";
+   
+    private static final String ACTIVE = "Active";
```

Comment:
In the future, we need to remove these hard-coded values and transfer them to a ProcurementCardStatus table